src/TextInputWithTokens.tsx

```diff
@@ -53,7 +52,7 @@ type TextInputWithTokensInternalProps<TokenComponentType extends AnyReactCompone
    * The number of tokens to display before truncating
    */
   visibleTokenCount?: number
-} & TextInputProps
+} & Omit<TextInputProps, 'size'>
```

Comment:
The `size` prop from `TextInputProps` was clashing with the `size` prop defined above